### PR TITLE
Modification to tree view to show the area of each location and the total for all locations

### PR DESCRIPTION
**Pull request by monroy95**
**Displaying default unit of measure in square meters:**

uom default

**Showing total square meters by location and total by all locations:**

tree view location

### CODE REVIEW

File: erpnext/assets/doctype/location/location_tree.js [hunk 1]
```diff
@@ -27,6 +27,44 @@ frappe.treeview_settings["Location"] = {
...
 		}
 	],
+	onrender: function (node) {
+
+		if (!node.is_root) {
+			frappe.db.get_value("Location", node.data.value, "area")
+				.then((r) => {
+					// Place the location area on the right side, tree accounts style.
+					$('<span class="balance-area pull-right text-muted small">'
+						+ (`${((flt(r.message.area)).toLocaleString('en') + ' Square Meters')}`)
```
Comment (rmehta) [inline erpnext/assets/doctype/location/location_tree.js]: Strings must be translatable

```js
_('{0} Sq m', flt(r.message.area).toLocaleString('en'));
```

(us abbreviation for narrow views)

**Review by rmehta — CHANGES_REQUESTED** [review]

File: erpnext/assets/doctype/location/location_tree.js (hunk 1 above)
Comment (monroy95) [reply erpnext/assets/doctype/location/location_tree.js]: Understood, I've applied the changes to show **Sq m** in 175838e

File: erpnext/assets/doctype/location/location_tree.js [hunk 2]
```diff
@@ -27,6 +27,44 @@ frappe.treeview_settings["Location"] = {
...
 		}
 	],
+	onrender: function (node) {
+
+		if (!node.is_root) {
+			frappe.db.get_value("Location", node.data.value, "area")
+				.then((r) => {
+					// Place the location area on the right side, tree accounts style.
+					$('<span class="balance-area pull-right text-muted small">'
+						+ (`${((flt(r.message.area)).toLocaleString('en') + ' Sq m')}`)
```
Comment (nabinhait) [inline erpnext/assets/doctype/location/location_tree.js]: Why hard-coded "Sq m"? As we already have a dedicated field "Area UOM".

**Review by nabinhait — CHANGES_REQUESTED** [review]

File: erpnext/assets/doctype/location/location_tree.js (hunk 2 above)
Comment (Tropicalrambler) [reply erpnext/assets/doctype/location/location_tree.js]: Was an oversight, we are aware it needs to be translatable and will make those changes!

Comment (Tropicalrambler) [reply erpnext/assets/doctype/location/location_tree.js]: We have made the changes expect a PR today.

**Pull request closed without merging**